plugins/chat/lib/chat_notifier.rb

```diff
@@ -124,7 +134,7 @@ def list_users_to_notify
 
   def chat_users
     users =
-      User.includes(:do_not_disturb_timings, :push_subscriptions, :user_chat_channel_memberships)
+      User.includes(:user_chat_channel_memberships, :group_users)
 
     users
       .distinct
```

Comment:
why do we split query  here ?

Reply:
Also why did you remove `:do_not_disturb_timings` and `:push_subscriptions` ?

Reply:
No need to split the query, will change. Those includes were added by mistake when I refactored this class, we don't need them.